/home/ubuntu/dev/lib/python3.10/site-packages/transformers/models/auto/configuration_auto.py:1020: FutureWarning: The `use_auth_token` argument is deprecated and will be removed in v5 of Transformers.
  warnings.warn(
/home/ubuntu/dev/lib/python3.10/site-packages/transformers/models/auto/auto_factory.py:472: FutureWarning: The `use_auth_token` argument is deprecated and will be removed in v5 of Transformers.
  warnings.warn(
Loading checkpoint shards:   0%|          | 0/3 [00:00<?, ?it/s]/home/ubuntu/dev/lib/python3.10/site-packages/transformers/modeling_utils.py:3728: ResourceWarning: unclosed <ssl.SSLSocket fd=113, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.0.224', 38156), raddr=('35.186.228.49', 443)>
  gc.collect()
ResourceWarning: Enable tracemalloc to get the object allocation traceback
Loading checkpoint shards: 100%|██████████| 3/3 [00:03<00:00,  1.04s/it]
/home/ubuntu/dev/lib/python3.10/site-packages/transformers/utils/hub.py:374: FutureWarning: The `use_auth_token` argument is deprecated and will be removed in v5 of Transformers.
  warnings.warn(
load INSTRUCTOR_Transformer
max_seq_length  512
/home/ubuntu/dev/lib/python3.10/site-packages/ray/serve/_private/utils.py:492: DeprecationWarning: setDaemon() is deprecated, set the daemon attribute instead
